src/pages/ProductDetail/Component/Cart.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { useNavigate } from 'react-router-dom'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faCashRegister } from '@fortawesome/free-solid-svg-icons'
+
+import './Cart.scss'
+
+const Cart = ({ isCartBtn, setIsCartBtn }) => {
+  return (
+    <div className="CartModal">
```

Comment:
```suggestion
    <div className="cart">
```
가장 부모 태그는 컴포넌트의 이름과 동일하게 하되
소문자로 시작하게 구현해주세요~